nss-crystal-cobras/BangazonSite
# Ticket #1 complete

Closes Ticket #1

Given a user is viewing any page on the Bangazon site
When the user performs a gesture on the Product Categories hyperlink
Then the user will see a view containing a list of all product categories
And next to the category name, the number of products in that category will be displayed
And the first three products in the category will be displayed beneath each category, that are hyperlinks to the product detail

Sample
Electronics (15)
    Dryer
    HD Television
    Sony Walkman

Sporting Goods (7)
    Football
    Ice skates
    Basketball hoop

Type of change
  New feature (non-breaking change which adds functionality)
  This change requires a documentation update

Testing Instructions
git fetch --all
git checkout CW-branch-one

On the Home page click on Product Categories in NavBar
You should see a list of all product categories with the number of products in that category.
Beneath the category the first 3 products will be displayed
Clicking on a product should route you to that product's details page

Checklist:
  [x] My code follows the style guidelines of this project
 [x] I have performed a self-review of my own code 
 [x] My changes generate no new warnings
